kodiak/main.py

```diff
@@ -131,6 +132,43 @@
     )
 
 
+def get_branch_name(raw_ref: str) -> Optional[str]:
+    """
+    Extract the branch name from the ref
+    """
+    if raw_ref.startswith("refs/heads/"):
+        return raw_ref.split("refs/heads/", 1)[1]
+    return None
+
+
+@webhook()
+async def push(push_event: events.PushEvent) -> None:
```

Comment:
I think it might be good to document either here with a doc string or somewhere else why we are setting up web hooks for the given event